auto_nag/scripts/duplicate_copy_metadata.py

```diff
@@ -138,6 +145,34 @@ def set_autofix(self, bug: dict, copied_fields: List[tuple]) -> None:
         autofix["comment"] = {"body": comment}
         self.autofix_changes[bug_id] = autofix
 
+    def get_previously_copied_fields(self, bug: dict) -> Set[str]:
```

Comment:
I wish we had a way without having to parse comments :(
We should think about that in the future.

Reply:
I agree, is fragile; I added some tests to mitigate that.